gtk-sys/src/gtk_glue.c

```diff
@@ -143,6 +143,10 @@ GtkTextTagTable* cast_GtkTextTagTable(void* widget) {
     return GTK_TEXT_TAG_TABLE(widget);
 }
 
+GtkTextIter* cast_GtkTextIter(void* widget) {
+    return GTK_TEXT_ITER(widget);
```

Comment:
How this passed travis is a mystery to me. I don't think there is a `GTK_TEXT_ITER` macro.

Anyway the `text_buffer` patch doesn't use `GTK_TEXT_ITER` so you can just remove this `cast_` business.